apps/aecore/include/txs.hrl

```diff
@@ -1,8 +1,4 @@
--record(signed_tx, {
-          data                       :: term(),
-          signatures = ordsets:new() :: ordsets:ordset(binary())}).
--type(signed_tx() :: #signed_tx{}).
-
+ %% opaque type for data structure signed_tx
```

Comment:
This comment is odd, types should not be in .hrl files so moving it is right but leaving this comment behind doesn't really help anyone, or?!